pagefaultgames/pokerogue
# [Bug] vite port (for development)

## What are the changes?


Setting the vite port to `8000`

## Why am I doing these changes?




it always was `8000` and wasn't supposed to be changed on purpose

## What did change?



vite port can now be configured and it `8000` by default

### Screenshots/Videos



## How to test the changes?





start the server and check the port.
If you want change the .env port to see it change

## Checklist
